### ISSUE TITLE
Accessibility issue with Radio Buttons

### ISSUE THREAD

**Issue by sagarjs** [Dogfalo/materialize#2191]
When i use the example given on the website, my screen reader does not read the radiobuttons. Neither is the checked state visible when using Windows High Contrast Mode

Example Code i used:

```
<form action="#">
    <p>
      <input name="group1" type="radio" id="test1" />
      <label for="test1">Red</label>
    </p>
    <p>
      <input name="group1" type="radio" id="test2" />
      <label for="test2">Yellow</label>
    </p>
    <p>
      <input class="with-gap" name="group1" type="radio" id="test3"  />
      <label for="test3">Green</label>
    </p>
      <p>
        <input name="group1" type="radio" id="test4" disabled="disabled" />
        <label for="test4">Brown</label>
    </p>
  </form>
```

I tried both in Firefox and IE. I am using NVDA screen reader on Windows 8

Please fix this issue, as it makes it impossible to use a radio button on my website

**Comment by obouchari** [comment]
Add some descriptive text for screen readers only, and assign a class of ".sr-only": 

.sr-only {
	position: absolute;
	width: 1px;
	height: 1px;
	padding: 0;
	margin: -1px;
	overflow: hidden;
	clip: rect(0,0,0,0);
	border: 0;
}

I hope this would help.

**Comment by sagarjs** [comment]
@obouchari the screen reader is not able to read the radio button, it does read the label associated with it though. So what i don't get is, whether the radiobutton exists, whether it is checked or unchecked. 

Since accessibility is a mandatory requirement by law in most countries, i guess fixing the accessibility issue will help Materialize be accepted by a much bigger audience. The switch is another example of a control which is not accessible. I love materialize controls and wish i could use them without having the accessibility issue.

**Comment by Dogfalo** [comment]
This used to be a problem when we had `visibility:hidden` applied to the radio button, but this is no longer the case and screen reader does read out the button.
